src/mlpack/core/data/imputation_methods/custom_imputation.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file custom_imputation.hpp
+ * @author Keon Kim
+ *
+ * Definition and Implementation of the empty CustomImputation class.
+ */
+#ifndef MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_IMPUTATION_HPP
+#define MLPACK_CORE_DATA_IMPUTE_STRATEGIES_CUSTOM_IMPUTATION_HPP
+
+#include <mlpack/core.hpp>
+#include <cmath>
```

Comment:
I think @zoq already pointed this out somewhere else, but `cmath` should already be included by `core.hpp`, so I think this line is unnecessary.